src/Pages/eunjinlee/Login/Login.js

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import './Styles/Common.scss'
+import './Login.scss'
+import { Link } from 'react-router-dom'
+
+const API = "http://3.35.19.3:8000"
+
+class LoginEunjinlee extends React.Component {
+
+  constructor() {
+    super();
+    this.state = {
+      id: "",
+      password: "",
+      hiddenPassword: true,
+      idError: false,
+      passwordError: false,
+    }
+  }
+
+  //update id and password value 
+  updateInputValue = (e) => {
+    const {id, value} = e.target
+    this.setState({
+      [id]: value //계산된속성명 
+    })
+  }
+
+  //show and hide password value
+  togglePassword = () => {
+    this.setState({
+      hiddenPassword: !this.state.hiddenPassword
+    })
+  }
+
+  //check if id is valid
+  checkValidId = () => {
+    if (!this.state.id.includes('@')) {
+      this.setState({
+        idError: !this.state.idError
+      })
+    }
+  }
+
+  //check if pw is valid
+  checkValidPassword = () => {
+    if (this.state.password.length >= 4) {
+      this.setState({
+        passwordError: !this.state.passwordError
+      })
+    }
+  }
```

Comment:
```suggestion
  checkValidPassword = () => {
    const isPWValid = this.state.password.length >= 4;
    
    this.setState({
        passwordError: isPWValid
    })
  }
```
- 이런식으로 boolean 데이터를 활용하면 좋습니다.
- 조건문은 위와 마찬가지로 불필요해 보입니다.